src/DIRAC/FrameworkSystem/private/authorization/AuthServer.py

```diff
@@ -51,35 +51,43 @@
 class AuthServer(_AuthorizationServer):
     """Implementation of the :class:`authlib.oauth2.rfc6749.AuthorizationServer`.
 
+    This framework has been changed and simplified to be used for DIRAC purposes,
+    namely authorization on the third party side and saving the received extended
+    long-term access tokens on the DIRAC side with the possibility of their future
+    use on behalf of the user without his participation.
+
+    The idea is that DIRAC itself is not an identity provider and relies on third-party
+    resources such as EGI Checkin or WLCG IAM.
+
     Initialize::
 
       server = AuthServer()
     """
 
     LOCATION = None
-    REFRESH_TOKEN_EXPIRES_IN = 24 * 3600
 
     def __init__(self):
-        self.db = AuthDB()
+        self.db = AuthDB()  # place to store session information
         self.log = log
         self.idps = IdProviderFactory()
-        self.proxyCli = ProxyManagerClient()
-        self.tokenCli = TokenManagerClient()
+        self.proxyCli = ProxyManagerClient()  # take care about proxies
+        self.tokenCli = TokenManagerClient()  # take care about tokens
+        # The authorization server has its own settings, but they are standardized
         self.metadata = collectMetadata()
         self.metadata.validate()
         # args for authlib < 1.0.0: (query_client=self.query_client, save_token=None, metadata=self.metadata)
-        # for authlib >= 1.0.0:
-        _AuthorizationServer.__init__(self, scopes_supported=self.metadata["scopes_supported"])
-        # Skip authlib method save_token and send_signal
-        self.save_token = lambda x, y: None
-        self.send_signal = lambda *x, **y: None
+        _AuthorizationServer.__init__(self, scopes_supported=self.metadata["scopes_supported"])  # for authlib >= 1.0.0
+        self.save_token = lambda x, y: None  # Skip original unused authlib method
+        self.send_signal = lambda *x, **y: None  # Skip original unused authlib method
```

Comment:
No, it because we have determined that DIRAC will not generate its own tokens, but will use the tokens obtained during authorization in third party identity providers. Therefore, some functions are not required in this implementation.

Ill add next comments:
```
        # authlib requires the following methods:
        # The following `save_token` method is called when requesting a new access token to save it after it is generated.
        # Let's skip this step, because getting tokens and saving them if necessary has already taken place in `generate_token` method.
        self.save_token = lambda x, y: None
        # Framework integration can re-implement the following method to support signal system.
        # But in this implementation, this system is not used.
        self.send_signal = lambda *x, **y: None
        ```

Reply:
added